### ISSUE TITLE
grml-live fails to build image 

### ISSUE THREAD

**Issue by ohausername** [grml/grml#82]
Running grml64-full 2017.05 Release Codename Freedatensuppe [2017-05-31] from an usb pen.

I wanted to build a grml image using the command 'live-build' following: http://grml.org/grml-live/#usage-examples, but it fails with:

...
GRMLBASE/96-apt-listbugs OK.
GRMLBASE/97-apt-listchanges OK.
GRMLBASE/98-clean-chroot OK.
GRMLBASE/99-finish-grml-build OK.
GRML_FULL/01-firefox OK.
Exit code task_configure: 420
Calling task_tests
WARNING: Subdirectory tests/ not found. No tests run.
Calling task_finish
Filesystem     Type      Size  Used Avail Use% Mounted on
Tue Nov 28 10:19:29 UTC 2017
The dirinstall took 2054 seconds.
Calling task_savelog
$LOGUSER is undefined. Not saving log files to remote.
Calling task_faiend
[ FAIL ] Error: there was a critical error during execution of stage 'fai dirinstall'
[ FAIL ] Note:  check out /mnt/sda/live/grml//grml_logs/fai// for details. [exit 6]
grml-live  313.49s user 90.46s system 19% cpu 34:20.67 total
root has logged off pts/0 from 192.168.1.239.
root has logged off pts/1 from tmux(3154).

Steps to reproduce

boot grml from usb pen and from a terminal I did:

tmux
mkdir /mnt/sda
mount /dev/sda1 /mnt/sda
mkdir /mnt/sda/live
cd /mnt/sda/live
grml-live

**Comment by mika** [comment]
You have to look into `/mnt/sda/live/grml//grml_logs/fai/` as noted in the error message, especially look for `shell.log`.

**Comment by ohausername** [comment]
I did take a log at those logfiles, but did not find anything suspicious. No obvious warnings or errors (but of course I could have missed something).

The shell.log ends like this:

...
Updating index cache for path `/usr/share/man/fr.UTF-8/man7'. Wait... 
Checking for stray cats under /usr/share/man/fr.ISO8859-1...
Checking for stray cats under /var/cache/man/fr.ISO8859-1...
Processing manual pages under /usr/share/man/fr.UTF-8...
Updating index cache for path `/usr/share/man/fr.UTF-8/man8'. Wait... 
done. 
Checking for stray cats under /usr/share/man/fr.UTF-8...
Checking for stray cats under /var/cache/man/fr.UTF-8...
Processing manual pages under /usr/local/man... 
86 man subdirectories contained newer manual pages. 
4778 manual pages were added. 
0 stray cats were added.
Cleaning /var/lib/clamav/ 
Setting up daily.cvd and main.cvd symlinks
Creating /etc/ld.so.nohwcap 
Skipping resolvconf workaround because FILE_RC class not in use.
Found /etc/mdadm/mdadm.conf, getting rid of any possible enabled ARRAY settings.
Warning: updatedb not installed 
Removing /etc/machine-id generated by systemd 
GRMLBASE/98-clean-chroot OK.
===== shell: GRMLBASE/99-finish-grml-build =====
GRMLBASE/99-finish-grml-build OK. 
===== shell: GRML_FULL/01-firefox ===== 
fcopy: copied /etc/grml/fai/config/files/etc/firefox-esr/firefox-esr.js/GRML_FULL to /mnt/sda/live/grml/grml_chroot/etc/firefox-esr/firefox-esr.js 
GRML_FULL/01-firefox OK.

**Comment by mika** [comment]
There must be something like `FAILED with exit code` or so in `shell.log`, in your initial bug report you also skipped the relevant stuff before `GRMLBASE/96-apt-listbugs OK.` where something failed.

Just upload your logfile / cmdline output to https://gist.github.com/ - with incomplete information nobody will be able to help you :)

**Comment by ohausername** [comment]
I didn't want to post the long logfiles here. Thanks for the tips about making a gist. I've created a gist here:
https://gist.github.com/ohausername/c9588bf4de6e83636145f2e7a773ac46
It contains the output from the terminal. Actually I did manage to build an iso file this time, by running 'grml-live' a second time, then it seems continue.

There is also the shell.log file with output from the first run of grml-live.
